drivers/filesystems/fastfat_new/allocsup.c

```diff
@@ -4702,7 +4702,9 @@ Return Value:
     if (Value != 0x1) {
 
         DebugTrace(+1, Dbg, "LogOf\n", 0);
+#ifdef FASTFATDBG  // __REACTOS__: Add '#ifdef'.
```

Comment:
NB: This addition is optional as long as `DebugTrace()` macro is defined as `NOTHING` when `FASTFATDBG` is undefined.